scarb/src/core/publishing/manifest_normalization.rs

```diff
@@ -38,6 +38,7 @@ pub fn prepare_manifest_for_publish(pkg: &Package) -> Result<TomlManifest> {
         package,
         workspace: None,
         dependencies,
+        dev_dependencies: None,
```

Comment:
Hmm, interestingly, Cargo seems to include dev deps in its packages: https://github.com/rust-lang/cargo/blob/3eb246cd875964b7ee33acbcf30e146ade7a11b9/src/cargo/util/toml/mod.rs#L276-L280

Not sure how we should do in Scarb though 🤔 I don't understand the rationale behind this in Cargo.

Reply:
@mkaput @tomek0123456789 Isn't this important to have it though?
What if pkg A has a test in `src/`, that depends on `alexandria` wired as `dev-dep`. Then pkg A can be published to registry, and downloaded as (normal) dependency of pkg B. When we want to run tests of B, the `cfg(test)` is enabled for all packages. Wouldn't we fail to compile tests of pkg B? 🤔 
Maybe this would be worth exploring with some experiment?

Reply:
Hmm, but we somehow do not run tests of dependencies anyway? How is this accomplished?

Because what I'd expect here is that no tests are compiled in alexandria 🤔

Reply:
@mkaput 

The other crates are not searched for tests. If they are compiled, is something I suggest we should verify. 

https://github.com/starkware-libs/cairo/blob/d4edafc66ead03ee67c86d53aad39fa48b525577/crates/cairo-lang-test-plugin/src/lib.rs#L52

Reply:
I generally agree, but I still think we should first verify if this very PR does not break something this works now, especially since it's not hard to verify with a quick test.

Reply:
It does not break anything because nobody is using this feature. If that's going to be a problem, users will just still use `[dependencies]` section.

Reply:
FYI Tomek has created an issue to tackle this later :D 
https://github.com/software-mansion/scarb/issues/940